libs/robotics-masters-robohatmm1-m4/ns.ts

```diff
@@ -0,0 +1,29 @@
+
```

Comment:
we've deprecated having custom ns.ts files, please remove.

Reply:
Sure.  No problem.  

Resolved by b6077cc